### PR TITLE
setup dependencies for server, db and client

### PR DESCRIPTION
**Pull request by dwwr**

### CODE REVIEW

File: client/.src/containers/app/AppStyles.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import styled from 'styled-components';
+
+const AppStyles = styled.div`
```
Comment (jwnukoski) [inline client/.src/containers/app/AppStyles.js:3]: You might benefit from putting your styles in a CSS file instead.

File: server/db/index.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+var mysql = require('mysql');
+
+var db = mysql.createConnection({
+    user: 'root',
```
Comment (jwnukoski) [inline server/db/index.js:11]: Supply the 'host' option here, if you're having problems connecting.

File: server/index.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+const express = require('express');
+const app = express();
+const port = 3000;
+
+app.use(express.static('./public'));
```
Comment (jwnukoski) [inline server/index.js]: You're in the server folder here. Relative path to public would be '../public'. Unless you plan on moving the folder.

**Review by jwnukoski — APPROVED** [review]
I've approved the changes.

Some things to check:
1. Make sure to check if you're DB connection is working properly. Specify a host, if it doesn't.
2. Verify your paths are working in server/index.js for the public folder.

File: client/.src/containers/app/App.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    super(props)
+    this.state = {}
+  }
+
+
+
+
+  render () {
+    return (
+      <AppStyles>Welcome to Guitar Centaur</AppStyles>
```
Comment (mmickeyd) [inline client/.src/containers/app/App.js:15]: What is the advantage to using styled components as opposed to css?

File: faker/seeder.js [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+    }
+  })
+
+
+
+}
+
+
+// console.log('randomItems:', randomItems.length);
+// console.log('randomImages:', randomImages.length);
```
Comment (mmickeyd) [inline faker/seeder.js:76]: There's a handful of commented out console.log's in this file that could be deleted now you know the seeder is working correctly.

File: faker/seeder.js [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+let randomItems = [];
+let randomImages = [];
+
+//populates arrays with random objs
+for (let i = 0; i < 100; i++) {
+  let itemObj = {
+    id: i,
+    itemNum: faker.random.number(),
+    POSNum: faker.random.number(),
+    avgScore: faker.random.number(5),
```
Comment (mmickeyd) [inline faker/seeder.js:15]: I'm just starting to do the seeding myself, so I'm not 100% if this is right or not, but I think you can set up a range of numbers. Relating to that, do you want a minimum value of 1 on both this and reviewCount below? Or do you want the random number range to also include 0?

File: server/db/index.js [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+const host  = 'localhost';
+const port  = 3000;
+const url   = `http://${host}:${port}`;
+
+
+
+var db = mysql.createConnection({
+    host: host,
+    user: 'root',
+    password: '',
```
Comment (mmickeyd) [inline server/db/index.js:12]: a password key-value pair isn't necessary when there's no password.

File: webpack.config.js [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+        test: /\.(?:ico|gif|png|jpg|jpeg)$/i,
+        type: 'asset/resource'
+      },
+      {
+        test: /\.(woff(2)?|eot|ttf|otf|svg|)$/,
+        type: 'asset/inline'
+      },
+      {
+        test: /\.(scss|css)$/,
+        use: ['style-loader', 'css-loader', 'postcss-loader']
```
Comment (mmickeyd) [inline webpack.config.js:31]: more or less, just asking this for my own personal understanding.. what are these tests for beyond the first one with babel-loader? I haven't seen this done before, so I'm just curious.

**Review by mmickeyd — COMMENTED** [review]
Looks really good dood! Honestly I couldn't find much, so I had to nitpick to find things. Your code looks really good though, well organized and clean!

**Pull request merged**